### PR TITLE
VC (and Quadruna FSM) tweaks

### PR DESCRIPTION
**Pull request by liam-ilan**
### Summary
Some minor tweaks to the VC, and a fix so that quadruna FSM can build with commit info

### Changelist 
- Added stackWaterMarkConfig to VC
- Added some canjson scaffolding to VC
- Added enque / tasks scaffolding to VC
- Fixed bug with commitinfo and quadruna FSM builds

### Testing Done
- Tests pass!

### Resolved Issues
None

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [x] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [x] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: firmware/quadruna/VC/test/test_temp.cpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+#include "test_vcBaseStateMachineTest.h"
+
+class FsmFaultsTest : public FsmBaseStateMachineTest
+{
+};
+
+TEST_F(FsmFaultsTest, temporary)
+{
+    ASSERT_TRUE(true);
+}
```
Comment (gtaharaedmonds) [inline firmware/quadruna/VC/test/test_temp.cpp:12]: nitpick: FSM -> VC

**Review by gtaharaedmonds — APPROVED** [review]
Looks perfect! Can you test this on the VC first? 
The hardware is now ready. Just verify that the program is stable (doesn't fault out anywhere during init) and that CAN TX works (with PCAN) would be sufficient.

File: firmware/quadruna/VC/test/test_temp.cpp (hunk 1 above)
Comment (liam-ilan) [reply firmware/quadruna/VC/test/test_temp.cpp]: fixed!

**Comment by liam-ilan** [comment]
Will mere after VC test

**Pull request merged**